sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslateContent.Serialization.cs

```diff
@@ -57,7 +57,7 @@ internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
                         continue;
                     }
 #if NET6_0_OR_GREATER
-				writer.WriteRawValue(global::System.BinaryData.FromStream(item));
```

Comment:
This doesn't compile, based on the else statement, this looks like the correct code. Please let me know if it's not.